docs/solidity-by-example.rst

```diff
@@ -645,99 +645,71 @@ Safe Remote Purchase
 Micropayment Channel
 ********************
 
-In this section we will learn how to build a simple implementation
-of a payment channel. It use cryptographics signatures to make
+In this section we will learn how to build an example implementation
+of a payment channel. It use cryptographic signatures to make
 repeated transfers of Ether between the same parties secure, instantaneous, and
-without transaction fees. To do it we need to understand how to
+without transaction fees. For the example, we need to understand how to
 sign and verify signatures, and setup the payment channel.
 
 Creating and verifying signatures
 =================================
 
 Imagine Alice wants to send a quantity of Ether to Bob, i.e.
 Alice is the sender and the Bob is the recipient.
+.. TODO: What does the below mean?
 Alice only needs to send cryptographically signed messages off-chain
 (e.g. via email) to Bob and it will be very similar to writing checks.
 
-Signatures are used to authorize transactions,
-and they are a general tool that is available to
-smart contracts. Alice will build a simple
-smart contract that lets her transmit Ether, but
-in a unusual way, instead of calling a function herself
-to initiate a payment, she will let Bob
-do that, and therefore pay the transaction fee.
+Alice and Bob use signatures to authorise transactions, which are a feature of smart contracts.
```

Comment:
Hm, verifying signatures is something that can be done in smart contracts on Ethereum, but I would not call it a feature of smart contracts.

Reply:
Clarified